_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,141 @@
+---
+Title: JShell Accumulo Feature
+Author: R. Dane Magbuhos
+Reviewers: Christopher Tubbs, Keith Turner
+---
+
+## Overview
+First introduced in Java 9+, [JShell][JShell Doc] is an interactive Read-Evaluate-Print-Loop (REPL) 
+Java tool that assess user's inputed declarations, statements, and expressions and outputs 
+the results. This tool provides a convenient way to test out and execute quick tasks with Accumulo
+in the terminal.
+
+## Major Features
+* During Accumulo build, produces a default JShell script called `jshell-init.jsh` 
+containing up-to-date [Accumulo Java APIs][public APIs] and [AccumuloClient][client] 
+build implementation
+
+* Startup JShell with default or custom JShell script 
+
+* Both JShell start up options automatically import all relevant Java APIs 
```

Comment:
By default they import classes to interact with the Accumulo API... but might not be all relevant Java APIs for the user's intended task. Also, this is only by default. The file is customizable by the user in their conf/ directory.